src/main/java/ArrayQueue.java

```diff
@@ -22,13 +22,14 @@ public ArrayQueue(int initialCapacity) {
     }
 
     @Override
-    public void enqueue(T element) {
+    public boolean enqueue(T element) {
```

Comment:
Tests also need updating to assert the new returned value.

Reply:
I'll do that in a subsequent PR that cleans up the implementations themselves.

Reply:
I'd rather see affected tests updated alongside the source changes.

Reply:
Added. 

Note that the tests for both queue implementations will be improved in subsequent PRs.